src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -216,7 +232,23 @@ private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootL
             try
             {
                 recursionDetector.Push(uid);
-                return TransformContentCore(_errors, definitions, file, rootSchema, schema, value, uidCount);
+                return TransformContentCore(
+                    _errors.With(e =>
+                    {
```

Comment:
extract a method and avoid the duplicated code in line 60

Reply:
OK.